ArcGISRuntimeSDKQt_QMLSamples/Maps/HonorMobileMapPackageExpiration/HonorMobileMapPackageExpiration.qml

```diff
@@ -78,38 +77,6 @@ Rectangle {
         }
     }
 
-    // Connect to the various signals on MobileMapPackageUtility
-    // to determine if direct read is supported or if an unpack
-    // is needed.
-    Connections {
-        target: MobileMapPackageUtility
-
-        onIsDirectReadSupportedStatusChanged: {
-            if (MobileMapPackageUtility.isDirectReadSupportedStatus !== Enums.TaskStatusCompleted) {
-                return;
-            }
-
-            // if direct read is supported, load the MobileMapPackage
-            if (MobileMapPackageUtility.isDirectReadSupportedResult) {
-                mmpk.load();
-            } else {
-                // direct read is not supported, and the data must be unpacked
-                MobileMapPackageUtility.unpack(mmpk.path, unpackPath)
```

Comment:
For some reason I can't add a comment to line 26 but the property "unpackPath" is no longer used so it can be removed.